### PR TITLE
replace some "word"s with "character"s in site/en/tutorials/text/text_generation.ipynb

### PR DESCRIPTION
**Pull request by masa-ita**
There are some "word"s in site/en/tutorials/text/text_generation.ipynb those should be "character"s, because the model used in this tutorial is character based text generator.

### CODE REVIEW

File: site/en/tutorials/text/text_generation.ipynb [hunk 1]
```diff
@@ -622,7 +638,7 @@
       "source": [
         "For each character the model looks up the embedding, runs the GRU one timestep with the embedding as input, and applies the dense layer to generate logits predicting the log-likelihood of the next character:\n",
         "\n",
-        "![A drawing of the data passing through the model](images/text_generation_training.png)"
+        "![A drawing of the data passing through the model](https://github.com/masa-ita/tf-docs/blob/fix_word_usage_in_site_en_tutorials_text_text_generation/site/en/tutorials/text/images/text_generation_training.png?raw=1)"
```
Comment (lamberta) [inline site/en/tutorials/text/text_generation.ipynb]: revert this URL change

File: site/en/tutorials/text/text_generation.ipynb [hunk 2]
```diff
@@ -1016,23 +1032,21 @@
...
         "\n",
-        "* The RNN state returned by the model is fed back into the model so that it now has more context, instead than only one word. After predicting the next word, the modified RNN states are again fed back into the model, which is how it learns as it gets more context from the previously predicted words.\n",
+        "* The RNN state returned by the model is fed back into the model so that it now has more context, instead than only one character. After predicting the next character, the modified RNN states are again fed back into the model, which is how it learns as it gets more context from the previously predicted characters.\n",
         "\n",
         "\n",
-        "![To generate text the model's output is fed back to the input](images/text_generation_sampling.png)\n",
+        "![To generate text the model's output is fed back to the input](https://github.com/masa-ita/tf-docs/blob/fix_word_usage_in_site_en_tutorials_text_text_generation/site/en/tutorials/text/images/text_generation_sampling.png?raw=1)\n",
```
Comment (lamberta) [inline site/en/tutorials/text/text_generation.ipynb]: revert

File: site/en/tutorials/text/text_generation.ipynb (hunk 1 above)
Comment (masa-ita) [reply site/en/tutorials/text/text_generation.ipynb]: Thank you for your advice.  The URL seems to be changed accidentally when I save the file from Colab.  Fixed.

File: site/en/tutorials/text/text_generation.ipynb (hunk 2 above)
Comment (masa-ita) [reply site/en/tutorials/text/text_generation.ipynb]: Fixed.

**Comment by lamberta** [comment]
Thanks @masa-ita 
This diff seems to be doing a bit more now: https://app.reviewnb.com/tensorflow/docs/pull/1202/files/
(Also, please download from Colab or adjust your local indentation setting for a cleaner diff.)

Closing this PR since the commits seem a bit messed up, but please submit another just only fixes the word => character issue.

**Pull request closed without merging**